openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/build/BuildConfigOperationsImpl.java

```diff
@@ -163,39 +162,10 @@ public Void trigger(WebHookTrigger trigger) {
   }
 
   @Override
-  public BuildConfigResource<BuildConfig, Void, Build> withResourceVersion(String resourceVersion) {
+  public BuildConfigOperationsImpl withResourceVersion(String resourceVersion) {
```

Comment:
Why switch to the implementation instead of the interface, here?

Reply:
I'll be making an update to remove that method altogether.  It is not needed because the base logic that calls newInstance will handle it.